definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -7014,5 +7014,47 @@
     "CaseEventID": "createGeneralOrder",
     "UserRole": "citizen",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
Not too strong just a thought may be we should update `LiveFrom` value for all the fields which are adding.